be/src/storage/lake/tablet_manager.cpp

```diff
@@ -267,7 +266,18 @@ SegmentPtr TabletManager::lookup_segment(std::string_view key) {
 void TabletManager::cache_segment(std::string_view key, SegmentPtr segment) {
     auto mem_cost = segment->mem_usage();
     auto value = std::make_unique<CacheValue>(std::move(segment));
-    fill_metacache(key, value.release(), (int)mem_cost);
+    fill_metacache(key, value.release(), mem_cost);
+}
+
+void TabletManager::update_segment_cache_size(std::string_view key) {
```

Comment:
looks like this is not only update the cache size but also redo the fill-metacache?

Reply:
yes, current lru cache does not support updating value size, so use refill to update.